tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -6621,13 +6613,45 @@ Status ConvertResize(OpConverterParams* params) {
   // Calculate output dimensions.
   // Given input dimensions [N, C, H, W] and output size [H_out, W_out],
   // output dimensions equals [N, C, H_out, W_out]
```

Comment:
This comment should go inside the if branch below

Reply:
The comment applies to both branches, as it's a similar logic in both cases: H_out and W_out are overridden by the scale attribute in both cases.

Reply:
I was thinking this comment was talking about calculating output_shape_dims, the else branch actually calculates a output shape tensor. Let's do this:
s/Calculate output dimensions./Calculate the output shape as static dimensions or a shape tensor:/

Reply:
Ah right, I guess keeping the word "dimensions" could be mis-interpreted given the two different methods there are now. I updated this.